README.md

```diff
@@ -34,14 +34,24 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. If you were to describe semantic HTML to the next cohort of students, what would you say?
 
+Semantic HTML is markup that describes what the content on a web page is. For example labeling a header or paragraph is semantic markup. Anything to do with styling is not, however some HTML elements do have styling built in.
+
 2. Name two big differences between ```display: block;``` and ```display: inline;```.
 
+display: block; creates a block-level element, whereas display: inline; creates an inline-level element. The block element will disrupt the flow of content on a page by adding space between itself and any other elements. Inline elements do not therefore they allow elements to stack next to each other. Both can be manipulated using box model properties like height and width.
```

Comment:
:exclamation: Be careful!

You _can't_ change the height and width of inline elements. If you need to have something behave inline, but need to manipulate its dimensions, you have to make it `display: inline-block;`